proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,128 @@
+# Cosmos Stargate Hub Upgrade Proposal 2
+
+## by Iqlusion
+
+October 31, 2020
+
+### Current Status: Draft
+
+Blocked on Final Code Hashes from the IBC, Tendermint and SDK teams.
+
+## Key Results
+
+------------
+If passed, this governance proposal would commit the Cosmos Hub to upgrading to halting the `cosmoshub-3` at 06:00 UTC on Nov 19 exporting the state and starting `cosmoshub-4` based on gaia 3.0.
+
+## Context
+
+------------
+
+In proposal 27, Iqlusion proposed a comprehensive process to translate the unprecented surface area of this upgrade. We called this process Stargate.
+
+Iqlusion is pleased to report that the Stargate Process has been successfully executed. We believe the chevrons are unlocked and that Hub can safely step throught the Stargate.
+
+After the success of the community approval of the first Stargate Upgrade proposal, we now follow-up with the second Stargate proposal for the Cosmos Hub that will outline the results of the first proposal and the process for upgrading the Hub.
+
+## The Stargate Plan
+
+------------
+The purpose of the Stargate effort was to ensure that Cosmos can fulfill the vision of an Internet of Blockchains in 2020 while mitigating the risks outlined above. It was also an attempt to create a process for complex future on-chain upgrades on the Cosmos Hub. That included the following steps:
```

Comment:
Reference to "risks outlined above" - presumably should be risks outlined in the original and add some link to it?

Reply:
We could also just remove this wording.

Reply:
I'd say its important to keep reference to risks as explicit as possible